evm/interchainTokenFactory.js

```diff
@@ -225,6 +226,64 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'registerCustomToken': {
```

Comment:
add coverage of new methods (custom token linking, flow limit set/get) in test-evm gh action so any issues can be caught in PRs

Reply:
I don't really know how to program those, we need to first deploy the token to register. Those actions are not tested for its either it seems because of their complexity.

Reply:
the `deployInterchainToken` already deploys a token from a hardcoded `tokenId`. The deployed token address can be used for custom token linking.

Reply:
example where we use a similar assumption
https://github.com/axelarnetwork/axelar-contract-deployments/pull/502/commits/0410347b04f49325e1c8c8bc849cd8dd6ffd7adf#diff-5c45946bd1812ebdac4c0960ab99634e76f7bdab539ef6b02d93c159267db822R133